gluten-ut/spark33/src/test/scala/io/glutenproject/utils/velox/VeloxSQLQueryTestSettings.scala

```diff
@@ -226,7 +226,8 @@ object VeloxSQLQueryTestSettings extends SQLQueryTestSettings {
     "typeCoercion/native/mapconcat.sql",
     "typeCoercion/native/mapconcat.sql",
     "typeCoercion/native/promoteStrings.sql",
-    "typeCoercion/native/stringCastAndExpressions.sql",
+    // Result different: Expected "{"a":[null]}", but got "{"a":[0]}"
+    // "typeCoercion/native/stringCastAndExpressions.sql",
```

Comment:
Could you help open an issue for this bug and put the link here?

Reply:
`Null` is not handled correctly for struct literal, adding below should fix this UT.
https://github.com/oap-project/gluten/pull/4267/files#diff-66c2133a2e9ec8f7a3c631d4aebb91e9e6e081f667b628e43e1754391aa2ec38R33-R35